ELLI/src/app/features/test/test.component.html

```diff
@@ -2,24 +2,25 @@
 <div class=category>
   <h2 *ngIf="isQuestionShowing"> {{gameDeck.cards[i].keyWord}}</h2>
 
+  <h1>Total: {{this.i+1}}/{{this.gameDeck.cards.length}} correct: {{this.correctCounter}} Incorrect: {{
```

Comment:
can you change this to a h2 or h3  tag . . . it's good practice to only have one h1 tag per website

Reply:
can you pull from main before resubmitting pull request